docs/content/en/docs/reference/changelog.md

```diff
@@ -6,6 +6,45 @@ weight: 35
 
 ## Unreleased
 
+## [v0.15.0](https://github.com/aws/eks-anywhere/releases/tag/v0.15.0)
+
+### Latest supported OS version details
+|              | vSphere | Baremetal |  Nutanix | Cloudstack | Snow  |
+|    :---:     |  :---:  |   :---:   |   :---:  |    :---:   | :---: |
+| Ubuntu       | 20.04	 | 20.04     |	20.04             | Not supported	     | 20.04 |  
+| Bottlerocket | 1.13.1  | 1.13.1    |  Not supported     | Not supported	     | Not supported   |
+| RHEL         | 8.7	   | 8.7	     |  Not supported     | 8.7	               | Not supported   |
+
+### Added
+- Kubernetes 1.26 support ([#5079](https://github.com/aws/eks-anywhere/issues/5079))
+- Workload clusters full lifecycle API support for Bare Metal provider ([#5237](https://github.com/aws/eks-anywhere/issues/5237))
+- IRSA support for Bare Metal ([#4361](https://github.com/aws/eks-anywhere/issues/4361))
+- Workload clusters full lifecycle API support for Nutanix provider ([#5190](https://github.com/aws/eks-anywhere/pull/5190))
+- OIDC support for Nutanix ([#4711](https://github.com/aws/eks-anywhere/pull/4711)
+- Registry mirror support for Nutanix ([#5236](https://github.com/aws/eks-anywhere/pull/5236))
+- Support for linking EKS Anywhere node VMs to Nutanix projects ([#5266](https://github.com/aws/eks-anywhere/pull/5266))
+- Add `CredentialsRef` to `NutanixDatacenterConfig` to specify Nutanix credentials for workload clusters ([#5114](https://github.com/aws/eks-anywhere/pull/5114))
+- Support for taints and labels for Nutanix provider ([#5172](https://github.com/aws/eks-anywhere/issues/5172))
+- Support for InsecureSkipVerify for RegistryMirrorConfiguration across all providers. Currently only supported for Ubuntu and RHEL OS. ([#1647](https://github.com/aws/eks-anywhere/issues/1647))
+- Support for configuring of Bottlerocket settings. ([#707](https://github.com/aws/eks-anywhere/issues/707))
+- Support for using a custom CNI ([#5217](https://github.com/aws/eks-anywhere/issues/5217))
+- Ability to configure NTP servers on EKS Anywhere nodes for vSphere and Tinkerbell providers ([#4760](https://github.com/aws/eks-anywhere/issues/4760))
+- Support for nonRootVolumes option in SnowMachineConfig ([#5199](https://github.com/aws/eks-anywhere/issues/5199))
+- Validate template disk size with vSphere provider using Bottlerocket ([#1571](https://github.com/aws/eks-anywhere/issues/1571))
+- Validate management cluster bundles version is the same or newer than bundle version used to upgrade a workload cluster([#5105](https://github.com/aws/eks-anywhere/issues/5105))
+- Set hostname for Bottlerocket nodes ([#3629](https://github.com/aws/eks-anywhere/issues/3629))
```

Comment:
Another item we need to add is support for mixed disks within the same node grouping for bare metal clusters.

https://github.com/aws/eks-anywhere/issues/3234